### PR TITLE
Add recipe for 'Gecos'

### PR DESCRIPTION
**Pull request by padix-key**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/gecos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: gecos
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/gecos/archive/v{{ version }}.tar.gz
+  sha256: 48fc10ae4d018531b21815bedb49b095b33e13394ec3a69266d28e7c7c4bc01e
+
+build:
+  number: {{ number }}
```
Comment (dopplershift) [inline recipes/gecos/meta.yaml:13]: Just avoid templating it and put it here directly:
```suggestion
  number: 0
```

File: recipes/gecos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% set version = "1.2.0" %}
+{% set number = "0" %}
```
Comment (dopplershift) [inline recipes/gecos/meta.yaml:2]: Can just remove the templating for number when using directly below:
```suggestion
```

Most recipes only templatize name and version.

File: recipes/gecos/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+source:
+  url: https://github.com/biotite-dev/gecos/archive/v{{ version }}.tar.gz
+  sha256: 48fc10ae4d018531b21815bedb49b095b33e13394ec3a69266d28e7c7c4bc01e
+
+build:
+  number: {{ number }}
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py < 37]
+  skip: true  # [osx]
```
Comment (dopplershift) [inline recipes/gecos/meta.yaml:16]: This looks like pure Python and could be noarch Python:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```
What made you need to skip the build on osx?

File: recipes/gecos/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - numpy >=1.13
```
Comment (dopplershift) [inline recipes/gecos/meta.yaml:25]: If going to noarch, need a Python version:
```suggestion
    - python >=3.7
    - pip
    - poetry
  run:
    - python >=3.7
    - numpy >=1.13
```

File: recipes/gecos/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - biotite >=0.21
+    - scikit-image >=0.17
+    - matplotlib >=3.0
```
Comment (dopplershift) [inline recipes/gecos/meta.yaml:28]: Unless you need a PyQt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base >=3.0
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/gecos/meta.yaml (hunk 3 above)
Comment (padix-key) [reply recipes/gecos/meta.yaml:16]: This is just a copy-paste error from another recipe of mine, thank you.

**Comment by padix-key** [comment]
Thank you for the review. I applied the suggested changes.

**Review by dopplershift — CHANGES_REQUESTED** [review]
The Windows build failed due to the command line program not being set up properly in the recipe. Try these changes.

**Comment by padix-key** [comment]
Now the Windows build fails, as it does not find a compatible version of the `jax` library. However `jax` is not a direct dependency of this library and also should not be a dependency somewhere in the dependency tree to my knowledge. Furthermore, the OSX and Linux builds seem not to install `jax`, at least I did not find any mention in the build logs. Am I missing something here?

**Review by dopplershift — APPROVED** [review]
Pending CI. Previous Windows failure was due to an unrelated recipe merged into staged-recipes.

**Pull request merged**